python/mirascope/llm/clients/anthropic/_utils/encode.py

```diff
@@ -206,7 +236,7 @@ def encode_request(
     if format is not None:
         if format.mode == "strict":
             raise FormattingModeNotSupportedError(
-                formatting_mode="strict", provider="anthropic"
+                formatting_mode="strict", provider=provider, model_id=model_id
```

Comment:
I left out model_id intentionally, as anthropic has no support for strict mode (ie. there is no model for which strict mode is supported). Adding model id to the message (imo) implies that there might be another model for which strict mode would be supported. If anthropic later adds strict support so that there is at least one valid choice, that would be a good time to include model_id here